docs/_docs/collections.md

```diff
@@ -95,6 +88,21 @@ written out to `<dest>/awesome/some_subdir/some_doc/index.html`.
   </p>
 </div>
 
+## Configuring permalinks for collections {#permalinks}
+
+You can customize the [Permalinks](../permalinks/) for your collection's documents by setting `permalink` property in the collection's configuration as follows:
+
+```yaml
+collections:
+  my_collection:
+    output: true
+    permalink: /awesome/:path/:title.output_ext
```

Comment:
`:title.:output_ext` here – there is a colon missing between the `.` and the `o`.